packages/javascript/bh-shared-ui/src/views/Explore/InfoStyles/Pane.tsx

```diff
@@ -38,7 +38,6 @@ const usePaneStyles = makeStyles((theme: Theme) => ({
         '& > div.node:nth-child(odd)': {
             background: theme.palette.neutral.tertiary,
         },
-        scrollbarGutter: 'stable',
```

Comment:
Components using this style were EdgeInfoPane and EntityInfoPanel. Both did not have a regression when moved. I believe this style was just an artifact from a previous design, so it should be safe to remove